app/jobs/rss_notifications_checker.rb

```diff
@@ -0,0 +1,59 @@
+require 'rss'
+require 'date'
+
+class RssNotificationsChecker < ApplicationJob
+  queue_as :notifications
+  attr_reader :url
+
+  after_perform do |job|
+    logger.info(_("Finished creating notifications for RSS feed: #{job.url}"))
+    self.class.set(:wait => 12.hours).perform_later
```

Comment:
This is what makes the task 'recurring'. Any ActiveJob task can call itself again after it finishes to create a 'recurring job'